### ISSUE TITLE
Custom freshclam.conf broken

### ISSUE THREAD

**Issue by DavidJFowler** [mko-x/docker-clamav#117]
**Describe the bug**
When using the FRESHCLAM_CONF_FILE environment variable to specify a custom freshclam.conf file the following error message is logged on start up:

```
[bootstrap] FRESHCLAM_CONF_FILE set, copy to /etc/clamav/freshclam.conf
[bootstrap] CLAMD_CONF_FILE set, copy to /etc/clamav/clamd.conf
[bootstrap] Checking for Clam DB in /var/lib/clamav/main.cvd
[bootstrap] Initial clam DB download.
WARNING: Ignoring deprecated option SafeBrowsing at /etc/clamav/freshclam.conf:18
ERROR: Parse error at /etc/clamav/freshclam.conf:21: Unknown option FILE
ERROR: Can't open/parse the config file /etc/clamav/freshclam.conf
```
Inspecting /etc/clamav/freshclam.conf shows that a `FILE` option has been appended to the file:

```
FILE /mnt/freshclam.conf
```

/mnt/freshclam.conf is the value of environment variable FRESHCLAM_CONF_FILE

**To Reproduce**
1. Add a volume holding a freshclam.conf. file
2. Set FRESHCLAM_CONF_FILE to point to the config file in the mounted volume

**Expected behavior**
ClamAV boots without error using the custom freshclam.conf

**Additional context**
Image mkodockx/docker-clamav:alpine-edge
Appears to be caused by this PR [112](https://github.com/mko-x/docker-clamav/pull/112),

**Comment by DavidJFowler** [comment]
Hi @mko-x ,
The revert appears to have fixed the issue.
I have configured my deployment to use image mkodockx/docker-clamav:1.0.0-alpine-edge as opposed to mkodockx/docker-clamav:alpine-edge
Will future releases to docker hub use a higher version number or am I still at risk of being affected by breaking changes on tag 1.0.0-alpine-edge?

Thanks

**Comment by eht16** [comment]
This is interesting.
There is a feature to replace exisiting configuration items with environment variables like `CLAMD_CONF_xxxx` (introduced in https://github.com/mko-x/docker-clamav/pull/31). If a configuration item does not exist yet, it is added to the file.
This is what happens here: the `FRESHCLAM_CONF_FILE` environment is added to the configuration file and the prefix is stripped, so `FILE xxx` remains.

The weird thing is the same happens for the Debian images and the `freshclam` process fails in the same way.
Is it possible that nobody used that feature with the Debian images and so it wasn't noticed? Or am I just doing something wrong?

@mko-x @DavidJFowler could you maybe double check?
E.g. 
```
docker run -it --rm -v `pwd`/freshtest.conf:/tmp/freshtest.conf -e FRESHCLAM_CONF_FILE=/tmp/freshtest.conf mkodockx/docker-clamav:buster-slim
```

**Comment by eht16** [comment]
Possible fix: https://github.com/mko-x/docker-clamav/pull/124

**Comment by mko-x** [comment]
Thank you very much @eht16 for providing this fix.

Fix is available in version 1.0.2 on docker hub
